webpack.rules.js

```diff
@@ -1,6 +1,9 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
 const path = require('path');
 const sass = require('sass');
+const createStyledComponentsTransformer = require('typescript-plugin-styled-components').default;
```

Comment:
just curious, what is the difference between the `webpack.config.js` file under the config folder with hte `webpack.main.js` file?

Reply:
i think its things not cleaned up after ejecting. One of my goals for later in the year is to clean up all this mess. I will also update the dependencies as a task later